1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,153 @@
+Feature: User edits Project
+# First three lines cover Why, Who, and What, following scenarios covers the How
+In order to document a project 
+As a logged in User or as a Curator
+I want to edit existing and create new projects
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User is logged in
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Save Button
+        And they see a Next Button
+
+    @xxx
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard"
+        # but this one enables a Curator to edit projects that he's not an
+        # participant of
```

Comment:
Stemmer det at eneste forskjell på de to scenarioene er at vi ikke skal liste opp prosjekter hvor man er Curator på My Projects?

Reply:
ser nå at du allerede har bekreftet dette i tidligere kommentar

Reply:
Curator rollen er ikkje virksom på "My Projects". User finner sine egne prosjekter på "My Projects". Ein Curator er også ein User, og finner kunn sine egne prosjekter under "My Projects".

Om Curator ønsker å opne eit prosjekt i wizard, så skjer det frå landingpage - jamfør Given i 
Scenario "Curator opens a Project in the Project Wizard". I linjen "And User is Curator on Project's Project Owner's Institution" definerer når Curator rollen gir edit tilgang på prosjektet.

Gir det bedre fornuft?